package.json

```diff
@@ -21,23 +21,26 @@
   "scripts-info": {
     "info": "List available script",
     "build_all": "Build all packages (ES6, CJS, UMD) and generate packages",
-    "build_cjs": "Build CJS package with clean up existing build, copy source into dist",
-    "build_es6": "Build ES6 package with clean up existing build, copy source into dist",
+    "build_cjs": "Build CJS package with clean up existing build",
+    "build_esm5": "Build ESM/ES5 package with clean up existing build",
+    "build_esm15": "Build ESM/ES2015 package with clean up existing build",
     "build_closure_core": "Minify Global core build using closure compiler",
     "build_global": "Build Global package, then minify build",
     "build_perf": "Build CJS & Global build, run macro performance test",
     "build_test": "Build CJS package & test spec, execute mocha test runner",
     "build_cover": "Run lint to current code, build CJS & test spec, execute test coverage",
-    "build_docs": "Build ES6 & global package, create documentation using it",
+    "build_docs": "Build ESM15 & global package, create documentation using it",
```

Comment:
should we just use "2015" instead of "15" throughout?

Reply:
@benlesh We switched to `esm2015` on the Angular project. Would you like to use the same?